linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -1198,5 +1202,23 @@ protected virtual void MarkInstruction (Instruction instruction)
 				break;
 			}
 		}
+
+		protected virtual bool ContinueWith (TypeDefinition type, TypeReference reference)
+		{
+			if (type == null) {
+				throw new ResolutionException (reference);
+			}
+
+			return true;
+		}
+
+		protected virtual bool ContinueWith (MethodDefinition method, MethodReference reference)
```

Comment:
Change this to be `HandleUnresolvedMethod (MethodReference reference)` and just throw